docs/DeveloperGuide.adoc

```diff
@@ -79,14 +80,19 @@ image::UiClassDiagram.png[]
 
```

Comment:
Diagram could be made less messier. Line should not overlap one another and should stay within the box.